### DISCUSSION TITLE
Status of project?

### DISCUSSION THREAD

**Discussion by rushowr** [fHDHR/fHDHR#336] [General]
Hello, I just wish to inquire about the project status. I see there's not been activity in some time, and just wanted to know if the project is abandoned or if there's just not been a need to make updates?

Thanks!

**Comment by deathbybandaid** [comment]
I'm proud of the work I did on this project, and I was hoping to build a community that would submit pull requests, but that never happened.

As it stands, there's no outstanding bugs that I'm aware of, and the base program still functions. The support for the origin plugins has always been a challenge to keep up with as site APIs are constantly changing, and I don't personally use fHDHR for more than tvheadend.

My life has grown complex, and the time I have to devote to working on side projects has gone down, with little benefit to my life to work on them.

**Reply by rushowr** [reply]
Completely understand mate, my life has had it's complexity increases to be sure. I don't fault ya, and I just wanted to see what's shakin' with the project.  Thanks for replying mate, and I may fork and work with things, possibly send pull requests. 

👍
